kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/streamable/KafkaTrackingToken.java

```diff
@@ -107,10 +110,11 @@ public static KafkaTrackingToken from(TrackingToken trackingToken) {
     /**
      * Retrieve the {@link TopicPartition}/offset {@link Map} stored in this {@link TrackingToken}. This collection
      * corresponds to the actual progress of a Consumer polling for records.
+     * JacksonSerializer require the method format in 'getField()" in the absence of certain serializer module.
```

Comment:
I would consider this line of text, although correct, to be unnecessary documentation whenever somebody looks up this operation. So, if you could remove it, that would be great.